common/lib/xmodule/xmodule/course_module.py

```diff
@@ -774,6 +774,30 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    learning_info = List(
+        display_name=_("Course Learning Information"),
+        help=_("Specify what student can learn from the course."),
+        default=[''],
+        scope=Scope.settings
+    )
+
+    instructor_info = Dict(
+        display_name=_("Course Instructor"),
+        help=_("Enter the details for Course Instructor"),
+        default={
```

Comment:
It's nice to see the format of the instructors dict here, but should the default value be an empty dict? Couldn't you end up rendering an instructor with no information?